src/Value/Lines.php

```diff
@@ -58,7 +58,9 @@ public function next(): void
 
     public function key(): int
     {
-        $this->validate();
+        if (!isset($this->array[$this->currentLine])) {
```

Comment:
I would prefer `array_key_exists()` instead of `isset()` (in general)

Reply:
`isset` is faster and the array holds only objects